pkg/builtinchecks/yamls/pdb.yaml

```diff
@@ -0,0 +1,11 @@
+name: "pod-disruption-budget"
+description: "Indicates when a DeploymentLike object do not have a \"Pod Disruption Budget\" specified provided it's replica count is more than the minimum specified value"
+remediation: "Create a \"Pod Disruption Budget\" in the specified namespace with a \"MaxUnavailableValue\" set equal to or more than the specified criteria."
+scope:
+  objectKinds:
+    - PodDisruptionBudget
+    - DeploymentLike
+template: "pod-disruption-budget"
+params:
+  minimumMaxUnavailableCriteria: 1
+  minReplicas: 1
```

Comment:
Should this work similar to how anti-affinity works?  IE if there is more then 1 replica then a PDB should be set.  If so there would be no need for a minReplica setting.